packages/block-library/src/image/image.js

```diff
@@ -83,9 +83,10 @@ const scaleOptions = [
 	},
 ];
 
-const disabledClickProps = {
+const disabledProps = {
 	onClick: ( event ) => event.preventDefault(),
 	'aria-disabled': true,
+	tabIndex: -1,
```

Comment:
>  was able to solve this problem by adding tabIndex="-1" to the a element of the image block, but are there any concerns about this?

This fixes the issue that @jeryj [reported](https://github.com/WordPress/gutenberg/pull/56043#issuecomment-1806337980) for me.

Seems to be acceptable according to the [spec](https://www.w3.org/TR/2011/WD-html5-20110525/editing.html#sequential-focus-navigation-and-the-tabindex-attribute), but I'll defer to other folks.

Reply:
@t-hamano No, you can't add `tabindex="-1"` like this. Doing that prevents the link from gaining focus with arrow key navigation. You need to be able to focus if you are going to hear things like the URL, ALT text, filename, etc.

Reply:
In the context of the editor, the link is a very recent addition whose only function is to ensure the editor styles match the frontend because some global styles target A-tags. The A-tag doesn't actually need an href value to achieve this.

Would it be acceptable to remove it from the accessibility tree or mark it as presentation only or something? 

Were the ALT text and filename not available from the IMG element previously? 

If getting these attributes is preferable and, bear with me as this might be wild 😄, could we add a data-href to the surrounding figure, which is the selectable "block" element?

Reply:
@ramonjd You can't add `role="presentation"` to the link or else you will also destroy the semantics of the `<img>` tag wrapped inside. I never actually knew you could link an image block so not sure how it worked before. Pretty sure if there is no link, you could not access that information because `<img>` by default cannot take focus. That is a bug that needs to be fixed somehow.

Reply:
Agreed. If you use your arrow keys to select the block and the `<a>` becomes focussed, you can always focus the block wrapper with left arrow. This is how all blocks with wrappers work with `tabindex="0"` set on the wrapper.

Reply:
Ah okay, I appreciate the explanation @alexstine Thank you! 👍🏻 

> I think we can discuss more broadly in another issue, such as what kind of keyboard interaction is expected in the editor and what should be communicated to screen readers when the block contains an a element or an img element. .

Sounds like a good approach.